compute_endpoint/globus_compute_endpoint/endpoint/identity_mapper.py

```diff
@@ -203,12 +203,17 @@ def identity_mappings(self):
         self.identity_mappings = None
 
     def map_identity(self, identity_set: t.Iterable[t.Mapping[str, str]]) -> str | None:
-        for mapper in self.identity_mappings:
-            for identity_data in identity_set:
+        for m_i, mapper in enumerate(self.identity_mappings, start=1):
+            for id_i, identity_data in enumerate(identity_set, start=1):
                 try:
                     identity = mapper.map_identity(identity_data)
                 except Exception as e:
-                    log.warning(f"Identity mapper failed -- ({type(e).__name__}) {e}")
+                    sub = identity_data.get("sub")
+                    log.warning(
+                        f"Identity mapper failed for mapper {m_i}"
```

Comment:
This is a lot more informative.  What do you think of adding the overall count of mapper/id to allow the admin to confirm the offending item(s) more clearly?
ie. 
`map_identity Identity mapper failed for mapper #1 (of 2) (ExternalIdentityMapping) with identity #2 (of 4) (abc91c2e-1234-4d38-9a8f-1cbf4a6ce811) -- (IdentityMappingError) Error mapping identity`

Reply:
I like it.  Just added.